Shell/Options.cpp

```diff
@@ -2191,9 +2188,9 @@ void Options::init()
     _literalComparisonMode.addProblemConstraint(mayHaveNonUnits());
     _literalComparisonMode.addProblemConstraint(notJustEquality());
 
-    _nonGoalWeightCoefficient = NonGoalWeightOptionValue("nongoal_weight_coefficient","nwc",1.0);
+    _nonGoalWeightCoefficient = NonGoalWeightOptionValue("nongoal_weight_coefficient","nwc"); // default 10.0 is hard-wired to the constructor
```

Comment:
Does this still need to be a class that does something weird, rather than `FloatOptionValue`?

Reply:
The code that uses the option actually expects two ints (which represent a ratio), but we pretend it's a float. 

I was just too lazy to touch it more. (And please appreciate that hiding the default and adding the comment was an improvement over the default there, whose change actually did not have any effect! ;))

Reply:
>whose change actually did not have any effect!

Oh. *Oh*. Much appreciated!